### ISSUE TITLE
Chart refresh parameter ignored

### ISSUE THREAD

**Issue by s0170071** [openhab/openhab-android#946]
### Actual behaviour
I have a group of number items (temperatures) being updated every minute. Below there is a chart of that group. The chart item is refreshed each along with the rest of the page if any number changes.

### Expected behaviour
The Chart Item should only be reloaded if the refresh time has elapsed. Latest openhab build does that correctly if the BasicUI is used in a browser.

 
### Steps to reproduce
1. define a group of numbers
2. update them in a regular interval.
3. put them in a frame inside a sitemap. 
4. add a chart of that group below, but inside that frame. Use large refresh parameter, e.g. 999999
5. view in browser, find that values refresh but chart does not. (=correct) 
6. view in android app, find chart refreshing with each value update. 

If you're mobile, a chart refresh can take several seconds making it impossible to view (before next refresh comes)

#### Client
* Android version: Oreo 
* Device model: Samsung S7
* Stock or customized ROM: no
* App version : 2.2.0
* Build flavor Full
* Device language: de

#### Server
* Server version:  >2.3.0 (latest unstable)
* Reverse Proxy: no
* Authentication method: none

#### openHAB Cloud
* openHAB Cloud version: none
* Self hosted cloud or myopenhab.org: none
* Authentication method : none

#### App log
```
where would I get that ?
```

#### openHAB Server log
```
log is clean, no warnings.
```

#### Reverse Proxy log
```
does not apply
```

**Comment by mueller-ma** [comment]
Sounds like a duplicate of #217

**Comment by s0170071** [comment]
Not sure about the dublicate. I experience a reload with the chart not being visible for a few seconds. 217 mentions the chart flashing.

**Comment by s0170071** [comment]
Seems fixed in 2.2.10 beta version.
